src/containers/GeneralInfo/GeneralInfo.tsx

```diff
@@ -1,13 +1,17 @@
 import React, {ReactElement, Dispatch, SetStateAction} from "react";
 import {NodeListContainer} from "../NodeList/NodeListContainer";
 import classNames from "classnames";
+import {useSelector} from "react-redux";
+import {RootState} from "../../app/rootReducer";
 
 interface IGeneralInfoProps {
     setElementsHidden: Dispatch<SetStateAction<boolean>>;
     areElementsHidden: boolean;
 }
 
 export const GeneralInfo = ({setElementsHidden, areElementsHidden}: IGeneralInfoProps): ReactElement => {
+    const node = useSelector((state: RootState) => state.node);
```

Comment:
I don't see how you handled the case if there is no node (node list is empty)? I can add a separate issue for that then with more UI details.

However, have in mind those edge cases.

Reply:
I handle it on new commit. 
App loads node list page by default(flow from the figma design), if nodeList is fetched, app shows statistics for the 1. one in the array.

If there are no nodes, arrow to show statistics is hidden and app displays "EmptyList" component.